### PR TITLE
(complib, app) Add icons to comp lib (and swap out one USB icon in the app)

### PR DESCRIPTION
**Pull request by mcous**
## overview

This PR copies the baby SVG icon component out of the app and into the components library. At the moment, the icon data are generated by:

1. Exporting the icon symbol as SVG from Sketch
2. Running the SVG through `svgo`
3. Copying the `path` and `viewBox` data from the optimized SVG

Obviously this needs to be automated at some point, but for now it works.

```js
import {Icon, WIFI} from '@opentrons/components'

export function SomeComponent () {
  return (
    <Icon name={WIFI} className='foobar' />
  )
}
```

Still TODO in a followup:

* Replace all icons in app with `Icon` component
* Remove old `Icon.js`, `icons.js`, and `icons.css` in app

This PR includes:

- [ ] Chore work
- [ ] Bug fixes
- [X] Backwards-compatible feature additions
- [ ] Breaking changes
- [X] Documentation updates
- [X] Test updates

## changelog

- (Feature) Added `Icon` to components library
- (Docs) Improved the components library README instructions
- (Docs) Added `Icon` to README
- (Tests) Added some `Icon` snapshot tests

## review requests

Really curious as to what y'all think about:

* README improvements
* Usage of Icon component and icon name constants
* How we're exporting the Icon component and name constants from the library

### CODE REVIEW

File: components/README.md [hunk 1]
```diff
@@ -2,40 +2,70 @@
...
+    * proxy `.css` imports to `identity-obj-proxy`
+    * transform `.js` files in `node_modules/@opentrons`
+
+### node/npm setup
+
+The root of the monorepo has an `.nvmrc` file with the correct Node version, so to make sure your Node and npm versions are correct:
+
+``` shell
+cd opentrons
+nvm use
```
Comment (Kadee80) [inline components/README.md]: 👍

**Comment by btmorr** [comment]
Testing on Windows. I did:

- `cd app && npm install`
- `cd ../components && npm install`
- `cd ../app && ENABLE_VIRTUAL_VIRTUAL_SMOOTHIE=true make dev`

Result:

comp_app_icons

**Comment by btmorr** [comment]
The Electron packaged app works on Windows, but `make dev` throws errors. My npm version is 5.6.0 and I did `make install` in /app and /components

comp_app_icons

**Comment by IanLondon** [comment]
Looks awesome. I think we should figure out why `make dev` is getting stuck on `type` not getting removed on Windows, but also I think we should merge this sooner than later b/c me & @Kadee80 are all touching `components/*` stuff. Should we put up an issue and merge?

**Comment by mcous** [comment]
Windows issues were a result of not being on `Node v8` and `npm v5.6.0`. We're now working on robustifying our Windows setup (and documentation) so we can avoid these problems in the future

**Pull request merged**